test/server/lib/oauth/model.test.ts

```diff
@@ -182,6 +183,7 @@ describe('server/lib/oauth/model', () => {
         authorizationCode: randStr(),
         expiresAt: new Date(),
         redirectUri: 'https://test.com',
+        scope: ['email'],
```

Comment:
Why have you added `scope` here?

Reply:
To validate below that the activity is logged correctly with the application and scopes authorized by the user